android/src/main/java/com/reactnativepagerview/PagerViewViewManager.kt

```diff
@@ -139,6 +145,13 @@ class PagerViewViewManager : ViewGroupManager<ViewPager2>() {
     viewPager.isUserInputEnabled = value
   }
 
+  @ReactProp(name = "initialPage", defaultInt = 0)
+  fun setInitialPage(viewPager: ViewPager2, value: Int) {
+    viewPager.post {
```

Comment:
It needs to be set within `post` because `ViewPager2` has an issue and initial page doesnt change unless this call is defered to the next cycle
more details here: https://stackoverflow.com/questions/56311862/viewpager2-default-position